scripts/release_scripts/initial_release_prep.py

```diff
@@ -0,0 +1,321 @@
+# limitations under the License.
+
+"""Script for initial release preparation."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+import os
+import re
+import sys
+
+import python_utils
+import release_constants
+from scripts import common
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_APPENGINE_PATH = os.path.join(
+    _PARENT_DIR, 'oppia_tools', 'google_appengine_1.9.67', 'google_appengine')
+_PYGSHEETS_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'pygsheets-2.0.2')
+_GOOGLE_PATH = os.path.join(_PYGSHEETS_PATH, 'google')
+
+sys.path.insert(0, _APPENGINE_PATH)
+sys.path.insert(0, _PYGSHEETS_PATH)
+
+# pylint: disable=wrong-import-position
+import google # isort:skip
+# pylint: enable=wrong-import-position
+
+# Refer: https://groups.google.com/forum/#!topic/google-appengine/AJJbuQ3FaGI.
+google.__path__.append(_GOOGLE_PATH)
+
+# pylint: disable=wrong-import-position
+import pygsheets # isort:skip
+# pylint: enable=wrong-import-position
+
+RELEASE_CREDENTIALS_FILEPATH = os.path.join(
+    os.getcwd(), 'oppia-release-credentials.json')
+# This file is created when pygsheets asks the user to authenticate.
+AUTH_FILEPATH = os.path.join(os.getcwd(), 'sheets.googleapis.com-python.json')
+
+
+def get_job_details_for_current_release(
+        job_details, job_name_header, month_header, author_email_header,
+        author_name_header, instruction_doc_header):
+    """Returns a list of job names for current release from a list
+    of job details.
+
+    Args:
+        job_details: list(dict). List of dictionary of job details.
+            The keys of each dict are the headers from the jobs
+            spreadsheet.
+        job_name_header: string. The header used for job names.
+        month_header: string. The header used for month of the release.
+        author_email_header: string. The header used for email address of
+            job author.
+        author_name_header: string. The header used for name of
+            job author.
+        instruction_doc_header: string. The header used for instruction
+            doc for running the job.
+
+    Returns:
+        list(dict). The list of job details for current release.
+    """
+    job_details_for_current_release = []
+    # First three letters of the release month. Eg: Nov, Dec etc.
+    current_release_month = datetime.datetime.utcnow().strftime('%h').lower()
+    for job_detail in job_details:
+        if current_release_month in job_detail[month_header].lower():
+            job_details_for_current_release.append({
+                'job_name': job_detail[job_name_header],
+                'author_email': job_detail[author_email_header],
+                'author_name': job_detail[author_name_header],
+                'instruction_doc': job_detail[instruction_doc_header]
+                })
```

Comment:
Deindent by 4 to match opening indent.

/cc @oppia/dev-workflow-team this should be a lint check.

Reply:
Done.